.github/renovate.json

```diff
@@ -4,16 +4,47 @@
   ],
   "baseBranches": [
     "main",
-    "/^release\\/v[0-9]+.x/",
-    "renovate-test"
+    "/^release\\/v[0-9]+.x/"
   ],
-  "prHourlyLimit": 2,
+  "prHourlyLimit": 12,
   "packageRules": [
     {
-      "matchBaseBranches": ["renovate-test"],
-      "matchDepNames": [
-        "kubernetes/kubernetes"
+      "matchDepNames": ["kubernetes/kubernetes"],
+      "separateMinorPatch": true,
+      "allowedVersions": ">=1.28.0"
```

Comment:
We could add a description here to call-out that the only purpose of `allowedVersions` is ensure that upstream range does not block patch bumps.

Reply:
👍